File: 2016/gara1/energia/att/energia.pas
function energia(K: int64): int64;
begin
    (* Mettete qui il codice della soluzione *)
    energia := 42;
end;


var
    K: int64;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, K);

    writeln(fw, energia(K));
    close(fr);
    close(fw);
end.

File: ed2/gara1/energia/att/energia.c
#include <stdio.h>
#include <assert.h>

long long int energia(long long int K) {
    // Mettete qui il codice della soluzione
    return 42;
}


int main() {
    FILE *fr, *fw;
    long long int K;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%lld", &K));

    fprintf(fw, "%lld\n", energia(K));
    fclose(fr);
    fclose(fw);
    return 0;
}
